next-cloudinary/.tsconfig

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Next.js",
```

Comment:
is this specific to the library? should this be Next Cloudinary?

Reply:
https://github.com/tsconfig/bases/blob/main/bases/next.json

If json allowed comments, I'd have stuck this in here. This is using the default from next...but the display option isn't actually something that's used by tsc at all, I don't think. So setting it to this was just me specifying where these options are coming from.